From cca77ce5e8c38ee163b82539be8050fc3fa9098d Mon Sep 17 00:00:00 2001
From: Mahesh Mahadevan <r9aadq@freescale.com>
Date: Wed, 26 Oct 2011 10:36:39 -0500
Subject: [PATCH 0499/2508] ENGR00160863 Update clocking code to support all
 UARTs

The MX6 Sabre-lite board uses UART2 for console. Add clock
code for this in the MX6 BSP

Signed-off-by: Mahesh Mahadevan <r9aadq@freescale.com>
---
 arch/arm/mach-mx6/clock.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-mx6/clock.c b/arch/arm/mach-mx6/clock.c
index 7ca3b88..f97617c 100644
--- a/arch/arm/mach-mx6/clock.c
+++ b/arch/arm/mach-mx6/clock.c
@@ -4611,6 +4611,9 @@ static struct clk_lookup lookups[] = {
 	_REGISTER_CLOCK(NULL, "emi_clk", emi_clk),
 	_REGISTER_CLOCK(NULL, "enfc_clk", enfc_clk),
 	_REGISTER_CLOCK("imx-uart.0", NULL, uart_clk[0]),
+	_REGISTER_CLOCK("imx-uart.1", NULL, uart_clk[0]),
+	_REGISTER_CLOCK("imx-uart.2", NULL, uart_clk[0]),
+	_REGISTER_CLOCK("imx-uart.3", NULL, uart_clk[0]),
 	_REGISTER_CLOCK(NULL, "hsi_tx", hsi_tx_clk[0]),
 	_REGISTER_CLOCK(NULL, "caam_clk", caam_clk[0]),
 	_REGISTER_CLOCK(NULL, "asrc_clk", asrc_clk[0]),
-- 
1.7.7.4

